case_gather/cron_jobs.py

```diff
@@ -6,7 +6,7 @@
 CRONLOGGER = logging.getLogger('cronJobs')
 
 class GatherCases(CronJobBase):
-    RUN_EVERY_MINS = 30
+    RUN_EVERY_MINS = 1
```

Comment:
Im working on dockerizing this project and ill be looking into the cronjobs very soon (adding gunicorn now, then working on cronjobs).

I was thinking of having that as an environmental variable, so we can just define it in our docker setup :)